app/manage.py

```diff
@@ -17,5 +19,5 @@ def main():
     execute_from_command_line(sys.argv)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
What's wrong with single quotes?

Reply:
Black did this. It prefers double apostrophes.